app/services/HBaseRestDataAccess.scala

```diff
@@ -0,0 +1,205 @@
+package services
+
+import javax.inject.Inject
+
+import com.netaporter.uri.dsl._
+import com.typesafe.scalalogging.LazyLogging
+import config.Properties
+import play.api.libs.ws.{ WSClient, WSResponse }
+import play.api.http.Status
+import play.api.Configuration
+import play.api.libs.json.{ JsArray, JsLookupResult, JsValue }
+import uk.gov.ons.sbr.models.units.{ Child, EnterpriseUnit, LEU, UnitLinks }
+import utils.Utilities._
+
+import scala.concurrent.{ Await, Future }
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+
+// TODO:
+// - when creating the childrenJSON, blocking code is used to resolve the Future, rather than
+//   using a Future within a case class, this may not be the best way to do it
+// - there are quite a few repeated .split("~").last, this could be put in a function
+
+/**
+ * Created by coolit on 05/02/2018.
+ */
+class HBaseRestDataAccess @Inject() (ws: WSClient, val configuration: Configuration) extends DataAccess with Properties with LazyLogging {
```

Comment:
Consider maybe moving some utility function (e.g. encoding stuff) to a separate Object